packages/cli/src/constants/constants.ts

```diff
@@ -29,6 +29,9 @@ export const CLIENT_LOGIN_URL = process.env.NOVU_CLIENT_LOGIN || 'https://web.no
 // github
 export const GITHUB_DOCKER_URL = 'https://github.com/novuhq/novu/tree/main/docker';
 
+// analytics
+export const SEGMENTS_WRITE_KEY = process.env.SEGMENTS_WRITE_KEY;
```

Comment:
How exactly are we going to inejct this write key? If I recall correctly there is no problem it will be public

Reply:
Ah, ok we can have it directly here then.